private/aws-echo-service/src/runtimeConfig.ts

```diff
@@ -41,5 +41,6 @@ export const getRuntimeConfig = (config: EchoServiceClientConfig) => {
       }),
     sha256: config?.sha256 ?? Hash.bind(null, "sha256"),
     streamCollector: config?.streamCollector ?? streamCollector,
+    userAgentAppId: config?.userAgentAppId ?? loadNodeConfig(NODE_APP_ID_CONFIG_OPTIONS),
```

Comment:
Since defaultUserAgent is deprecated, the call on line 33 should be changed to createDefaultUserAgentProvider.

Reply:
will do that codegen change to clients in a separate PR

Reply:
https://github.com/aws/aws-sdk-js-v3/pull/6558